main.star

```diff
@@ -4,6 +4,7 @@ DEFAULT_SCRAPE_INTERVAL = "15s"
 
 def run(plan, 
         metrics_jobs=[], 
+        name="prometheus",
```

Comment:
lets try to add new optional arguments at the end!

https://app.circleci.com/pipelines/github/kurtosis-tech/awesome-kurtosis/1693/workflows/ce3742c0-e93c-4562-86d7-c1c3461de8c1/jobs/4354

breaks stuff like this otherwise